jwql/tests/test_setup_info.py

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python
+"""Tests for the ``setup.py`` module.
+
+Authors
+-------
+
+    - Bryan Hilbert
+
+
+Use
+---
+
+    These tests can be run via the command line (omit the ``-s`` to
+    suppress verbose output to ``stdout``):
+
+    ::
+
+        pytest -s test_setup_info.py
+"""
+import pytest
```

Comment:
@bhilbert4 I don't think this import is actually needed afterall

Reply:
Removed. I reran the test and it still passes.